rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -5,20 +5,15 @@
 #include "Emu/RSX/GSRender.h"
 #include "Emu/IdManager.h"
 #include "Emu/Cell/ErrorCodes.h"
+#include "Emu/Cell/Modules/cellGcmSys.h"
```

Comment:
yes, I'm including this to initial the gcm offset table in hle with my testcase, since it relies on a combination of sys_rsx syscalls and and cellgcm calls.

Reply:
though maybe I should remove this since no game would use a mix of cellgcm and sys_rsx calls like that.